File: TreinoProva/simulado_atividade/lib/model/PessoaModel.dart
class PessoaModel {
  int id = 0;
  String nome = "";
  String email = "";
  String senha = "";

  PessoaModel();
  PessoaModel.full(this.nome, this.email, this.senha);
  PessoaModel.fromJson(Map<String, dynamic> json)
    :
      id = json["id"],
      nome =  json["nome"],
      email = json["email"],
      senha = json["senha"]
    ;

  int get getId => this.id;

 set setId(int id) => this.id = id;

 String get getNome => this.nome;

 set setNome(String nome) => this.nome = nome;

  get getEmail => this.email;

 set setEmail( email) => this.email = email;

  get getSenha => this.senha;

 set setSenha( senha) => this.senha = senha;


}

File: TreinoProva/SimuladoAPI/src/main/java/com/example/SimuladoAPI/model/PessoaModel.java
package com.example.SimuladoAPI.model;

import jakarta.persistence.*;

@Entity
@Table (name = "Pessoa")
public class PessoaModel {


    @Id
    @GeneratedValue (strategy = GenerationType.IDENTITY)
    private int id;
    @Column
    private String nome;
    @Column
    private String email;
    @Column
    private String senha;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getSenha() {
        return senha;
    }

    public void setSenha(String senha) {
        this.senha = senha;
    }
}
